feature/google/src/test/java/com/revenuecat/purchases/google/BillingWrapperTest.kt

```diff
@@ -848,13 +849,21 @@ class BillingWrapperTest {
                         get() = emptyList()
                     override val tags: List<String>
                         get() = emptyList()
+                    override val presentedOfferingIdentifier: String?
+                        get() = null
                     override val purchasingData: PurchasingData
                         get() = purchasingData
                 }
             override val purchasingData: PurchasingData
                 get() = purchasingData
+            override val presentedOfferingIdentifier: String?
+                get() = null
             override val sku: String
                 get() = id
+
+            override fun copyWithOfferingId(offeringId: String): StoreProduct {
+                return this // this is wrong, just doing for test
```

Comment:
:homer_hiding:

Reply:
sigh. felt like overkill since this test doesn't use that. but i should refactor this to use our normal stubs anyway